docs/www/guide-wasm-elastic.md

```diff
@@ -1,261 +1,285 @@
+      record.headers.some( header =>
+        header.headerKey.equals(Buffer.from(key)) &&
+        header.value.equals(Buffer.from(value))
+      )
+
+    /* Transform function */
+    transform.processRecord((recordBatch) => {
+      console.log("Applying Wasm process function")
+      return Promise.all(
+        recordBatch.records.map((record) => {
+          if (containHeader("logService", "elastic", record)) {
+            console.log("Index record value to Elasticsearch")
+            return client.index({
+              index: "result_wasm",
+              type: "redpanda_wasm",
+              body: {
+                /*
+                index the record value into Elasticsearch
+                */
+                record_message: record.value.toString()
+              }
+            })
           }
         })
-      }
-    })
-  ).then(() => {
+      ).then(() => {
+          /*
+          return a map with recordBatch indexed to Elasticsearch
+          */
+          const result = new Map();
+          result.set("elasticIndex", recordBatch)
+          /* 
+          Wasm processRecord function has to return a Map always,
+          although that map is empty map
+          */
+          return result
+      })
       /*
-       return a map with recordBatch indexed to Elasticsearch
+        Note: if the previous promise failed, the Wasm errorHandler is
+        going to handle that error. You can use `deregister policy` to 
+        remove this Wasm script from the Wasm engine.    
       */
-      const result = new Map();
-      result.set("elasticIndex", recordBatch)
-      /* 
-       Wasm processRecord function has to return a Map always,
-       although that map is empty map
-      */
-      return result
-  })
-  /*
-    Note: if the previous promise failed, the wasm errorHandler is
-    going to handle that error, in this case, Deregister policy will
-    remove this wasm script from wasm engine.    
-  */
-});
+    });
+
+    exports["default"] = transform;
+    ```
 
-exports["default"] = transform;
-```
+6. Bundle the project files into a single file with:
 
-after applying the new logic, we need to bundle everything into a single file.
-For it, we can use 
+    ```bash
+    #this command install all nodejs dependecies
+    $ npm install
+    #this command generate `dist/elastic.js` file.
+    $ npm run build
+    ```
 
-```bash
-#this command install all nodejs dependecies
-$ npm install
-#this command generate `dist/elastic.js` file.
-$ npm run build
-```
+## Deploy the Wasm function to Redpanda
 
-Before deploying this Wasm script, we need to create the `origin` topic
-The statically defined input topic that this Wasm operator has defined in its metadata
+Now we need to deploy the Wasm script so that it can process the transforms.
 
-```bash
-rpk topic create origin
-```
+1. Create a topic to use for the transform:
 
- ## Deploy the Wasm function to Redpanda
+    ```bash
+    rpk topic create origin
```

Comment:
I think we should explicitly state how this topic is directly correlated to the wasm functions subscriptions list

Reply:
it a good idea, something like "this topic is gonna use for wasm subscription"